nuget/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.Package.csproj

```
@@ -9,6 +9,7 @@
     <Summary>Analyzers for .NET Compiler Platform ("Roslyn")</Summary>
     <ReleaseNotes>Diagnostic analyzers for the Microsoft .NET Compiler Platform (Roslyn)</ReleaseNotes>
     <PackageTags>Roslyn CodeAnalysis Compiler CSharp VB VisualBasic Diagnostic Analyzers Syntax Semantics</PackageTags>
+    <PackageReadmeFile>documentation\readme.md</PackageReadmeFile>
```

NIT: it might doesn't matter but it seems uses just `README.md`
```suggestion
    <PackageReadmeFile>README.md</PackageReadmeFile>
```

---

We can do so, but there are also other documentation files in the package, like `Microsoft.CodeAnalysis.Analyzers.md`. We already package them within the `documentation` folder. Additionally, it's the same layout in the existing `Microsoft.CodeAnalysis.NetAnalyzers` package: https://github.com/dotnet/roslyn-analyzers/blob/3587540ae9de2561eb00d2ee182e4d0b8c170d30/nuget/NetAnalyzers/Microsoft.CodeAnalysis.NetAnalyzers.Package.csproj#L16

---

OK, never mind